src/middlewared/middlewared/plugins/tunables.py

```diff
@@ -58,79 +73,101 @@ async def tunable_type_choices(self):
 
     @accepts(Dict(
         'tunable_create',
+        Str('type', enum=TUNABLE_TYPES, default='SYSCTL', required=True),
         Str('var', required=True),
         Str('value', required=True),
-        Str('type', enum=TUNABLE_TYPES, default='SYSCTL', required=True),
-        Str('comment'),
+        Str('comment', default=''),
         Bool('enabled', default=True),
         register=True
     ))
     async def do_create(self, data):
         """
-        Create a Tunable.
+        Create a tunable.
+
+        If `type` is `SYSCTL` then `var` is a sysctl name (e.g. `kernel.watchdog`) and `value` is its corresponding
+        value (e.g. `0`).
+
+        If `type` is `UDEV` then `var` is an udev rules file name (e.g. `10-disable-usb`, `.rules` suffix will be
+        appended automatically) and `value` is its contents (e.g. `BUS=="usb", OPTIONS+="ignore_device"`).
         """
         verrors = ValidationErrors()
+
         if await self.middleware.call('tunable.query', [('var', '=', data['var'])]):
-            verrors.add('tunable.create', f'Tunable {data["var"]!r} already exists in database.', errno.EEXIST)
+            verrors.add('tunable_create.var', f'Tunable {data["var"]!r} already exists in database.', errno.EEXIST)
 
-        if data['var'] not in await self.middleware.call('tunable.get_system_tunables'):
-            verrors.add('tunable.create', f'Tunable {data["var"]!r} does not exist in kernel.', errno.ENOENT)
+        if data['type'] == 'SYSCTL':
+            if data['var'] not in await self.middleware.call('tunable.get_sysctls'):
+                verrors.add('tunable_create.var', f'Sysctl {data["var"]!r} does not exist in kernel.', errno.ENOENT)
 
         verrors.check()
 
-        data['orig_value'] = await self.middleware.call('tunable.get_or_set', data['var'])
-
-        if (comment := data.get('comment', '').strip()):
-            data['comment'] = comment
+        data['orig_value'] = ''
+        if data['type'] == 'SYSCTL':
+            data['orig_value'] = await self.middleware.call('tunable.get_sysctl', data['var'])
 
-        _id = await self.middleware.call(
+        id = await self.middleware.call(
```

Comment:
Why change variable name that overwrites a builtin?

Reply:
That's what we do everywhere